run_one_experiment.py

```diff
@@ -40,9 +40,12 @@
 DEBUG: bool = False
 
 # Default LLM hyper-parameters.
-# WARN: Avoid large NUM_SAMPLES in highly parallelized local experiment.
-# NUM_SAMPLES controls the number of LLM responses per query, which may exceed
-# your LLM's limit on query-per-second.
+# - #182 shows Gemini returns NUM_SAMPLES independent responses via repeated
+#  queries, which generally performs better than top-k responses from one
+#  query [1].
+# WARN: please set the value according to your own LLM's query-per-second
+#     limit, especially in highly parallelized local experiment.
+# [1] TODO(@happy-qop): Update the link.
```

Comment:
Ah, sorry, I should have been clearer. Would it be better to move `[1]` right after your new comment? E.g.,
```python
# #182 shows Gemini returns NUM_SAMPLES independent responses via repeated
#  queries, which generally performs better than top-k responses from one
#  query [1].
# [1] TODO(@happy-qop): Update the link.
# WARN: Avoid large NUM_SAMPLES in highly parallelized local experiments. 
# It controls the number of LLM responses per prompt, which may exceed your 
# LLM's limit on query-per-second.
```

Sorry about the confusion!

Reply:
Thanks, merged!